modules/rosa-installing-prerequisites.adoc

```diff
@@ -73,11 +74,11 @@ $ aws ec2 describe-regions
 |+-----------------------------------+-----------------------+-----------------+|
```

Comment:
Can we change the command to use instead of `aws ec2 describe-regions` can we use `aws sts get-caller-identity` - step 4 in my doc.

Reply:
Done in a new JIRA: https://issues.redhat.com/browse/OSDOCS-1802